pandas/tests/groupby/test_groupby.py

```diff
@@ -2758,6 +2758,27 @@ def test_tuple_correct_keyerror(self):
         with tm.assert_raises_regex(KeyError, "(7, 8)"):
             df.groupby((7, 8)).mean()
 
+    def test_cython_with_timestamp_and_nat(self):
+        # https://github.com/pandas-dev/pandas/issues/19526
+        ts = pd.Timestamp('2016-10-14 21:00:44.557')
+        df = pd.DataFrame({'a': [0, 1], 'b': [ts, pd.NaT]})
+        index = pd.Int64Index([0, 1], dtype='int64', name='a')
```

Comment:
you can write the expected like
``expected = DataFrame({'b': [ts, NaT]}, index=Index([0, 1, name='a'))``

note we don't use ``pd.`` anywhere (the imports are at the top)